sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -3,7 +3,10 @@
 ## 12.17.0 (Unreleased)
 
 ### Features Added
-
+- The `get_blob_client` API now returns a version-aware `BlobClient`, in which all APIs that accept `version_id` will operate on the
+version ID provided to the `get_blob_client` API call by default.
```

Comment:
Maybe change the wording here a little to say we added a keyword that _can_ return a version-aware client. Something like this perhaps:

> Added optional keyword `version_id` to `get_blob_client` APIs which, if provided, will result in a version-aware `BlobClient` in which all APIs that accept `version_id` will operate on the version ID provided to the `get_blob_client` API call by default.

Nit: Maybe also consider putting this update second in the list.

Reply:
Will do, thanks for the feedback 😄 I do like the more careful verbiage you've suggested here